### ISSUE TITLE
can't re-define globals

### ISSUE THREAD

**Issue by selfsame** [bakpakin/Fennel#213]
Previous versions of fennel allowed this:

```
Welcome to Fennel 0.4.0-dev!
Use (doc something) to view documentation.
>> (global x 1)

>> (global x 2)
Compile error: Compile error in 'global' unknown:2: global x conflicts with local
>>
```

**Comment by jaawerth** [comment]
I believe this was caused by the fix introduced to resolve #197 merged in **0.3.1**. Because the global sym is beingintroduced to `scope.manglings`, it's likely confusing that defined symbol for a local.

I *think* there will be an easy fix here, but if all else fails we can add a `scope.globalManglings` or something like that...

**Comment by technomancy** [comment]
Yep, definitely an issue with the fix for #197. I'll put a fix together.

**Comment by technomancy** [comment]
You can pull in master or do a quick workaround by using `set-forcibly!`.
